docs/docs/guides/labs/components/components-etl-pipeline-tutorial.md

```diff
@@ -80,7 +80,7 @@ When you run commands like `dg list component-type` , `dg` obtains the results b
 
 ### 2. Confirm availability of the Sling component type
 
-To confirm that the `dagster_components.sling_replication` component type is now available, run the `dg list component-type` command again:
+To confirm that the `dagster.components.sling_replication` component type is now available, run the `dg list component-type` command again:
```

Comment:
should be "`dagster_sling.SlingReplicationCollectionComponent` component type ..." (this was wrong before this PR)